magic-links/24hour.js

```diff
@@ -95,13 +95,13 @@ jQuery(document).ready(function($) {
     }
     let week_day_names = days_for_locale(navigator.language, 'narrow')
     let headers = `
-      <div class="day-cell week-day">${week_day_names[0]}</div>
-      <div class="day-cell week-day">${week_day_names[1]}</div>
-      <div class="day-cell week-day">${week_day_names[2]}</div>
-      <div class="day-cell week-day">${week_day_names[3]}</div>
-      <div class="day-cell week-day">${week_day_names[4]}</div>
-      <div class="day-cell week-day">${week_day_names[5]}</div>
-      <div class="day-cell week-day">${week_day_names[6]}</div>
+      <div class="day-cell week-day" style="color:black;font-size:12px;font-weight:550;margin-bottom:5px;">S</div>
```

Comment:
@prykon we don't want `${week_day_names[0]}` to be replaced

Reply:
lets make this css a class too :)

Reply:
Done and done. The css class is declared in 24hour.php